src/stylesheets/core/_functions.scss

```diff
@@ -809,74 +815,36 @@ Gets a valid USWDS flex value
 
 /*
 ----------------------------------------
-font()
+scale()
 ----------------------------------------
-Gets
-- 'family'
-- 'size'
-- 'weight'
-based on on props passed
+Get a normalized typescale from a family
+and scale
 ----------------------------------------
 */
 
-@function font($props...){
-  $type: smart-quote(nth($props, 1));
-  @if $type == 'family' {
-    $our-family: smart-quote(nth($props, 2));
-    @if not map-has-key($project-font-stacks, $our-family){
-      @error "#{$our-family} is not a valid font family. Valid values: #{map-keys($project-font-stacks)}";
-    }
-    @elseif not map-get($project-font-stacks, $our-family) {
-      @error "Font family `#{$our-family}` is disabled in your project's theme settings. Set its value to `true` to use this family.";
-    }
-    @else {
-      @return map-get($project-font-stacks, $our-family);
-    }
+@function scale($props...){
```

Comment:
Removed the `family` and `weight` options from this function. Vestigial functionality — converted to single-use (typescale only).